pkg/config/setting.go

```diff
@@ -297,6 +298,10 @@ func ParseSetting(secrets, volCtx map[string]string, options []string, usePod bo
 		jfsSetting.Envs = env
 	}
 
+	if secrets["extraPreStopHookCmd"] != "" {
+		jfsSetting.ExtraPreStopHookCmd = secrets["extraPreStopHookCmd"]
```

Comment:
can we simply use `prestop` for simplicity?

the thing is ... we already have 2 types of naming conventions in this json struct (`secret-key|envs_map`), can't help to feel reluctant to add a third `extraPreStopHookCmd`

Reply:
what about `extra-prestop-cmd`

user needs to be know that this is an **extra** cmd and not an overwrite?

Reply:
> user needs to be know that this is an extra cmd and not an overwrite?

this is a bit like helm values, where all fields are in fact "extra", but they do not have the "extra" word as prefix, like `annotations`, `labels`

we'll introduce the minor detail in the docs, but even if users misunderstood and treat it like an overwrite, it won't hurt anything, correct?